testing/kuttl/e2e/additional-volumes/files/00-cluster-created.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: postgres-operator.crunchydata.com/v1beta1
+kind: PostgresCluster
+metadata:
+  name: additional-vols
+status:
+  instances:
+    - name: instance1
+      readyReplicas: 1
+      replicas: 1
+      updatedReplicas: 1
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: additional-vols-primary
```

Comment:
While the operator is obviously not responsible for creating the PVCs specified in `00-create-cluster.yaml`, shouldn't we still assert that they have indeed been created before moving to the next step in the kuttl test?